R/carehomes.R

```diff
@@ -186,10 +187,18 @@ NULL
 ##'   rate is used for all age groups; if a vector of values if used it should
 ##'   have one value per age group.
 ##'
-##' @param model_pcr_and_serology_user A value of 1 or 0 so switch on or off the
+##' @param model_pcr_and_serology A value of 1 or 0 so switch on or off the
 ##'   flows out of T_PCR_neg and T_sero_neg and the corresponding cap on the
 ##'   number of individuals leaving the R compartments
 ##'
+##' @param model_super_infection A value of 1 or 0 so switch on or off
```

Comment:
could do this later but would be good to add some examples showing how to set parameters for the multi strain model